CHANGES.rst

```diff
@@ -66,6 +66,7 @@ solarsystem.pds
 ^^^^^^^^^^^^^^^
 
 - New module to access the Planetary Data System's Ring Node System. [#2358]
+- Rename from ringnode to ring-moon systems node [#2900]
```

Comment:
No need for changelog as this only affects the dev version (I can remove the commit before merging, no todo on your side)